sal/class.json-api-site-base.php

```diff
@@ -395,7 +395,7 @@ function get_admin_url() {
 	}
 
 	function get_unmapped_url() {
-		return get_site_url( $this->blog_id );
+		return get_site_url( get_current_blog_id() );
```

Comment:
Yes since $this->blog_id is the .com blog id and it doesn't the right value was giving me a php notice. Since that database doesn't exist. 
```
WordPress database error Table 'enej_wp.wp_69984173_options' doesn't exist for query SELECT option_value FROM wp_69984173_options WHERE option_name = 'wp_69984173_user_roles' LIMIT 1 made by wp_xmlrpc_server->serve_request, IXR_Server->IXR_Server, IXR_Server->__construct, IXR_Server->serve, IXR_Server->call, IXR_Server->multiCall, IXR_Server->call, call_user_func, Jetpack_XMLRPC_Server->json_api, WPCOM_JSON_API->serve, WPCOM_JSON_API->process_request, call_user_func_array, WPCOM_JSON_API_GET_Site_Endpoint->callback, WPCOM_JSON_API_GET_Site_Endpoint->build_current_site_response, WPCOM_JSON_API_GET_Site_Endpoint->render_response_keys, WPCOM_JSON_API_GET_Site_Endpoint->render_response_key, WPCOM_JSON_API_GET_Site_Endpoint->render_option_keys, SAL_Site->get_unmapped_url, get_site_url, switch_to_blog, WP_Roles->reinit, get_option
```